pkgs/development/libraries/qt-6/modules/qtdeclarative.nix

```diff
@@ -0,0 +1,15 @@
+{ qtModule
+, qtbase
+, qtshadertools
+, openssl
+, python3
+}:
+
+qtModule {
+  pname = "qtdeclarative";
+  qtInputs = [ qtbase qtshadertools ];
+  propagatedBuildInputs = [ openssl python3 ];
+  preConfigure = ''
+    export LD_LIBRARY_PATH="$PWD/build/lib''${LD_LIBRARY_PATH:+:}$LD_LIBRARY_PATH"
```

Comment:
Why is this necessary? I'm skeptical that this isn't masking other problems.

Reply:
> That's frustrating: our CMake setup is supposed to handle that. slightly_frowning_face

Would you mind point me some direction for fixing that?